src/components/File.jsx

```diff
@@ -69,7 +67,7 @@ class File extends Component {
       this.props.onOpen(id)
     } else {
       // TODO Handle files opening throught the app instead of doing it throught the browser
-      window.open(`${STACK_URL}/files/download/${id}`, '_blank')
+      window.open(`${cozy._url}/files/download/${id}`, '_blank')
```

Comment:
Can you show me where? I can't figure how this can be defined, and more surely how it can work in both `dev` and `production` mode…